src/Kunstmaan/MenuBundle/AdminList/MenuItemAdminListConfigurator.php

```diff
@@ -38,7 +38,6 @@ public function __construct(EntityManager $em, AclHelper $aclHelper = null, Menu
     public function buildFields()
     {
         $this->addField('title', 'Title', false, 'KunstmaanMenuBundle:AdminList:menu-item-title.html.twig');
-        $this->addField('online', 'Online', false, 'KunstmaanMenuBundle:AdminList:menu-item-online.html.twig');
```

Comment:
In my opinion it is confusing to have a online / offline value for an external link.
Also, I feel that the online offline value should only be shown on the pages and not on the menu.